### ISSUE TITLE
Duplicate export when function exported under default and its name

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [webpro/knip#228]
Coming over from https://github.com/JoshuaKGoldberg/astro-konamimojisplosion/pull/13, given a file like:

```ts
export function konamimojisplosion() {
	return { /* ... */ }
}

export default konamimojisplosion;
```

Knip reports that the export is duplicated:

```plaintext
> knip

Duplicate exports (1)
konamimojisplosion, default  src/index.ts
```

That duplicate export is intentional though. I'd wanted to export under both styles so folks can use either import style for convenience.

Can we go with either or both of:

* This kind of behavior on my end is common+reasonable enough that it shouldn't be reported by default?
* Having an option to toggle whether this report would be made in that case?

**Comment by webpro** [comment]
My thought is that, in order to remove unused exports, it's best to be explicit. That's why duplicate exports are reported by default.

You can use `--exclude duplicates` (or `"exclude": ["duplicates"]`) to exclude this issue type from the report.

**Comment by JoshuaKGoldberg** [comment]
I do want to still have duplicates detection though. Could we have a way to remove it just for this one case? Like a per-file or per-line or per-name disable?

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.22.0. See [Release 2.22.0](https://github.com/webpro/knip/releases/tag/2.22.0) for release notes._

**Comment by webpro** [comment]
Added `@alias` to ignore instances of duplicate exports!
